src/components/TouchableWithoutFocus.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {TouchableOpacity} from 'react-native';
+import PropTypes from 'prop-types';
+const propTypes = {
```

Comment:
Up to you. Sorry last suggestion was incorrect.

```suggestion
import PropTypes from 'prop-types';

const propTypes = {
```

Reply:
Fixed